backend/common/src/main/java/ai/verta/modeldb/common/artifactStore/storageservice/s3/S3Controller.java

```diff
@@ -51,7 +51,7 @@ public ResponseEntity<UploadFileResponse> storeArtifact(
 
   @GetMapping(value = {"${artifactEndpoint.getArtifact}/{FileName}"})
   public ResponseEntity<Resource> getArtifact(
-      @PathVariable(value = ModelDBConstants.FILENAME) String fileName,
+      @PathVariable(value = CommonConstants.FILENAME_QUERY_PARAMETER) String fileName,
```

Comment:
this isn't a query parameter here, is it? It's a path parameter? I think this making "Filename" a "common" constant is a mistake. It's used in too many ways, and changing one of them would inadvertently break the others.

Reply:
Fixed